src/pages/trader-tools/common/_style.ts

```diff
@@ -6,6 +6,16 @@ import { Button, LinkButton } from 'components/form'
 import device from 'themes/device'
 import Patterns from 'images/common/dmt5-signals/dmt5-signals-patterns.png'
 
+type CalculatorTabItemType = {
+    active?: React.ReactNode
+    disabled?: React.ReactNode
+}
+
+type PnlCalculatorTabItemType = {
+    active?: React.ReactNode
+    disabled?: React.ReactNode
+}
+
```

Comment:
These should be props.
```suggestion
type CalculatorTabItemProps = {
    active?: React.ReactNode
    disabled?: React.ReactNode
}

type PnlCalculatorTabItemProps = {
    active?: React.ReactNode
    disabled?: React.ReactNode
}

```